torch/jit/batchop.py

```diff
@@ -101,11 +110,145 @@ def batch_where(data, mask, dims, data1, mask1, dims1, data2, mask2, dims2):
     res_dims = dims1 or dims2
     return res_data, res_mask, res_dims
 
+
+@torch.jit.script
+def batch_update(batch_data, batch_mask, batch_dims, new_data, new_mask, new_dims):
+    data = torch.where(new_mask, new_data, batch_data)
+    return data, new_mask, new_dims  # TODO: consider whether return new_mask and new_dims
+
+
+@torch.jit.script
+def batch_any(data, mask, dims):
+    return torch.gt(torch.sum(data * mask), 0)
+
+
+@torch.jit.script
+def batch_type_as(data, mask, dims, data1, mask1, dims1):
+    return data.type_as(data1), mask, dims
+
+
+@torch.jit.script
+def batch_gt(data, mask, dims, data1, mask1, dims1):
+    return torch.gt(data, data1), mask * mask1, dims or dims1
+
+
+@torch.jit.script
+def batch_size(data, mask, dims, dim):
+    return data.size(dim)
+
+
+@torch.jit.script
+def batch_argmax(data, mask, dims, dim, keepdim):
+    # if dim == 0:
```

Comment:
Add a TODO comment here specifying that we want to have this check functionality at some point